ledger/testing/consensusRange.go

```diff
@@ -69,38 +91,32 @@ var consensusByNumber = []protocol.ConsensusVersion{
 // created and inserted in consensusByNumber. At that point, your feature is
 // probably active in that version. (If it's being held in vFuture, just
 // increment your `start`.)
-func TestConsensusRange(t *testing.T, start, stop int, test func(t *testing.T, ver int, cv protocol.ConsensusVersion)) {
+func TestConsensusRange(t *testing.T, start, stop int, test func(t *testing.T, ver int, cv protocol.ConsensusVersion, cfg config.Local)) {
 	if stop == 0 { // Treat 0 as "future"
 		stop = len(consensusByNumber) - 1
 	}
 	require.LessOrEqual(t, start, stop)
+	cfg := config.GetDefaultLocal()
 	for i := start; i <= stop; i++ {
-		var version string
-		if i == len(consensusByNumber)-1 {
-			version = "vFuture"
-		} else {
-			version = fmt.Sprintf("v%d", i)
-		}
-		t.Run(fmt.Sprintf("cv=%s", version), func(t *testing.T) {
-			test(t, i, consensusByNumber[i])
+		version := versionStringFromIndex(i)
+		disable := rand16(t)%2 == 0
+		t.Run(fmt.Sprintf("cv=%s,LRU-cache-disable=%t", version, disable), func(t *testing.T) {
+			cfg.DisableLedgerLRUCache = disable
+			test(t, i, consensusByNumber[i], cfg)
 		})
 	}
 }
 
 // BenchConsensusRange is for getting benchmarks across consensus versions.
-func BenchConsensusRange(b *testing.B, start, stop int, bench func(t *testing.B, ver int, cv protocol.ConsensusVersion)) {
+func BenchConsensusRange(b *testing.B, start, stop int, bench func(t *testing.B, ver int, cv protocol.ConsensusVersion, cfg config.Local)) {
 	if stop == 0 { // Treat 0 as "future"
 		stop = len(consensusByNumber) - 1
 	}
+	cfg := config.GetDefaultLocal()
```

Comment:
Not sure what's happening here. I don't really think you need to widen the `BenchConsensusRange` signature - it's not important to run benchmarks without the cache.

And then it seems like you assign to `cfg` anyway. So remove the extra argument?

Reply:
good point, I will just remove these args and revert related changes.

Reply:
changes in [5954351](https://github.com/algorand/go-algorand/pull/5058/commits/5954351c1d37d56708629e2dd3d3d10f889fa162)